client/entrypoints/information-overlay/__tests__/index.test.js

```diff
@@ -0,0 +1,46 @@
+import { useDispatch } from '@wordpress/data';
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import InformationOverlay from '../information-overlay';
```

Comment:
The file being tested is `information-overlay`, can the file name on this test reflect that (instead of `index.test.js`)?